.github/workflows/performance-test.yml

```
@@ -0,0 +1,516 @@
+        ./contrib/install-prometheus-infra.sh
+    
+    - name: Upload Prometheus installation logs
+      if: always()
+      uses: actions/upload-artifact@v4
+      with:
+        name: prometheus-install-logs
+        path: prometheus-install.log
+        retention-days: 7
+
+    - name: Download and setup kube-burner v2.1.0
+      run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:4:54: Double quote to prevent globbing and word splitting

(shellcheck)